docs/use-cases/data-processing.md

```diff
@@ -7,3 +7,7 @@
 ## Videos
 
 * [Running a Data Replication Pipeline on Kubernetes with Argo and Singer.io](https://towardsdatascience.com/running-a-data-replication-pipeline-on-kubernetes-with-argo-and-singer-io-2fab5b0bad?utm_source=argo-docs)
+
+## Books
+
+* [Distributed Machine Learning Patterns](https://github.com/terrytangyuan/distributed-ml-patterns)
```

Comment:
since we have the ML page, should this only be on that page? Whereas this page would be more focused on more "pure" data work (e.g. ETLs)?

Reply:
There's a dedicated chapter in the book for data processing.

Reply:
Could you specify the chapter here? Something like "(see Chapter 12 on Data Processing)"

That makes it more clear why this one is in this list specifically (since we do have separate lists).

Reply:
Done